### ISSUE TITLE
RuboCop spaces -> tab indentation generates overly indented output.

### ISSUE THREAD

**Issue by ioquatix** [rubocop/rubocop#13120]
I'm using the following somewhat minimal configuration:

```
AllCops:
  DisabledByDefault: true

Layout/IndentationStyle:
  Enabled: true
  EnforcedStyle: tabs

Layout/InitialIndentation:
  Enabled: true

Layout/IndentationWidth:
  Enabled: true
  Width: 1

Layout/IndentationConsistency:
  Enabled: true
  EnforcedStyle: normal

Layout/EndAlignment:
  Enabled: true
  EnforcedStyleAlignWith: start_of_line

Layout/BeginEndAlignment:
  Enabled: true
  EnforcedStyleAlignWith: start_of_line

Layout/ElseAlignment:
  Enabled: true

Layout/DefEndAlignment:
  Enabled: true

Layout/CaseIndentation:
  Enabled: true

Layout/CommentIndentation:
  Enabled: true

Layout/EmptyLinesAroundClassBody:
  Enabled: true

Layout/EmptyLinesAroundModuleBody:
  Enabled: true

Style/FrozenStringLiteralComment:
  Enabled: true
```

And I'm applying it to this code, which apparently has no issues:

<img width="773" alt="image" src="https://github.com/user-attachments/assets/0990dc6e-066d-45fd-98b8-f1c9392d96c5">

```ruby
# frozen_string_literal: true

describe Protocol::Redis::Methods::Scripting do
	describe '#script' do
		describe "#debug" do
					let(:mode) {"YES"}

					it "can generate correct arguments" do
							expect(object).to receive(:call).with("SCRIPT", "DEBUG", mode)

							object.script("DEBUG", mode)
						end

					it "will convert subcommand symbols to strings" do
							expect(object).to receive(:call).with("SCRIPT", "debug", mode)

							object.script(:debug, mode)
						end
				end
	end
end
```

This code was actually originally generated from code which used spaces for indentation, but RuboCop generated the above conversion (which is not really correct).

In other words, given this input:

```
# frozen_string_literal: true

describe Protocol::Redis::Methods::Scripting do
  describe '#script' do
    describe "#debug" do
      let(:mode) {"YES"}

      it "can generate correct arguments" do
        expect(object).to receive(:call).with("SCRIPT", "DEBUG", mode)

        object.script("DEBUG", mode)
      end

      it "will convert subcommand symbols to strings" do
        expect(object).to receive(:call).with("SCRIPT", "debug", mode)

        object.script(:debug, mode)
      end
    end
  end
end
```

With the above configuration, it generates poorly indented output.

**Comment by koic** [comment]
Can you add the configuration to enable `Layout/BlockAlignment` in your `.rubocop.yml`?

```console
# .rubocop.yml
Layout/BlockAlignment:
  Enabled: true
```

For more details about this configuration, please refer to https://docs.rubocop.org/rubocop/1.65/cops_layout.html#layoutblockalignment.

**Comment by ioquatix** [comment]
Okay, I'll try it, thanks :)

**Comment by ioquatix** [comment]
Okay, I can confirm, the code is generated correctly with that change.

**Comment by ioquatix** [comment]
Thanks for your quick reply :)

**Comment by koic** [comment]
You are welcome! It's good to hear that the issue has been resolved :-)

**Comment by ioquatix** [comment]
Well, it was user error :)
